client/header/activity-panel/panels/reviews.js

```diff
@@ -162,7 +162,7 @@ class ReviewsPanel extends Component {
 					</Fragment>
 				);
 			} else {
-				buttonUrl = getAdminLink( 'edit-comments.php' );
+				buttonUrl = getAdminLink( 'edit-comments.php?comment_type=review' );
```

Comment:
Eh? Any reason why this change is in this PR?

Reply:
Totally a mistake. I'll fix.